class.jetpack-gutenberg.php

```diff
@@ -87,6 +85,20 @@ private static function prepend_block_prefix( $block_name ) {
 		return 'jetpack/' . $block_name;
 	}
 
+	/**
+	 * Remove the 'jetpack-' prefix from an extension name
```

Comment:
Just a nitpick: this is actually removing the `jetpack/` (with a slash) prefix instead of `jetpack-` (with a dash)

Reply:
Good spot. It's actually used for both blocks (`jetpack/`) and plugins (`jetpack-`).
I'll update the JSDoc. (I plan to make the prefix entirely explicit in a later PR.)

Reply:
According to the code this is removing only the `jetpack/` prefix, so it seems we're not considering the plugins here (unless I'm missing something obvious).

Should we then update the function to handle both of them?

```
return preg_replace( '/jetpack[\/-]/', '', $extension_name);
```

Reply:
Continued here: https://github.com/Automattic/jetpack/pull/11212#discussion_r254748101